sites/main-page-title/index.json-ld

```diff
@@ -354,11 +418,29 @@
 			],
 			"predecessor": "_:iteration_mpt_2"
 		},
+		{
+			"@id": "_:iteration_mpt_4",
+			"name": "Main page title aligned on the right",
+			"date": "2024-09",
+			"detectableBy": "h1#wb-cont[dir='rtl'], [dir='rtl'] h1#wb-cont",
+			"additions": [
+				"Added right to left support"
+			]
```

Comment:
Please also add `"successor": "_:iteration_mpt_4"` to the Iteration 3.

```suggestion
			],
			"predecessor": "_:iteration_mpt_3"
```

Reply:
added. Thanks